bin/pt-online-schema-change

```diff
@@ -11495,6 +11501,15 @@ value of C<10000>.
 
 The tool prints a warning and continues if a variable cannot be set.
 
+=item --sleep
+
+type: float; default: 0
+
+Interval of time (in seconds) to wait after copying a chunk of rows to the new 
+table. Use this if you are having trouble on servers with heavy load and other 
+alternatives are not viable: See  L<"--max-load">, <"--max-lag">. 
```

Comment:
To fix some punctuation and avoid "you" (which, afaik, PT docs generally do), I'd write this as:
"""
How long to sleep (in seconds) after copying each chunk. This option is useful when throttling by L<"--max-lag"> and L<"--max-load"> are not possible. A small, sub-second value should be used, like 0.1, else the tool could take a very long time to copy large tables.
"""

Ideally, it'd be nice to document why max lag and load aren't possible. Users sometimes ask "When would/should I use this?"